pkg/webhook/admission/http.go

```diff
@@ -34,6 +34,24 @@ import (
 var admissionScheme = runtime.NewScheme()
 var admissionCodecs = serializer.NewCodecFactory(admissionScheme)
 
+// adapted from https://github.com/kubernetes/kubernetes/blob/c28c2009181fcc44c5f6b47e10e62dacf53e4da0/staging/src/k8s.io/pod-security-admission/cmd/webhook/server/server.go
+//
+// From https://github.com/kubernetes/apiserver/blob/d6876a0600de06fef75968c4641c64d7da499f25/pkg/server/config.go#L433-L442C5:
+//
+//	     1.5MB is the recommended client request size in byte
+//		 the etcd server should accept. See
```

Comment:
Given all the stuff people are doing today. Can we really still assume that controller-runtime is only used in environments where etcd is used?

Reply:
Probably not, but this limiter should only apply to admission webhooks; which seems fair?

Reply:
I assume the apiserver calls admission webhooks independent of what datastore is used behind it.

I wasn't thinking about non-apiserver scenarios. More about the cases where folks are just not using etcd as storage

Reply:
Nevermind. I missed that every object that is stored goes through the apiserver which enforces the limit